permits/forms.py

```diff
@@ -1403,7 +1403,10 @@ def clean(self):
                 )
             )
 
-        if not cleaned_data["parent_{}".format(cleaned_data["document_type"].pk)]:
+        if not self.cleaned_data.get("document_type"):
+            raise ValidationError(_("Un type doit être renseigné!"))
```

Comment:
The requirement of `document_type` is already validated at this point, so you can just:

`return cleaned_data`